client/jobs.lua

```diff
@@ -81,46 +78,58 @@ local function startWork()
     StopAnimTask(cache.ped, "anim@gangops@facility@servers@", "hotwire", 1.0)
 end
 
-local function canInteractWithBox(i)
-    return InJail and CurrentJob and not isWorking and i == currentBox
+local function canInteractWithTask(i)
+    return InJail and prisonJob and not isWorking and i == currentTask
+end
+
+function chooseJob(job)
+    prisonJob = job
+    jobLocations = newRandomArray(#Config.Jobs[prisonJob].locations)
 end
 
--- Threads
+function getPrisonJob()
```

Comment:
This can just be directly passed as a function to the export, no need to create a new variable for it